packages/clay-date-picker/src/DateNavigation.tsx

```diff
@@ -126,8 +126,9 @@ const ClayDatePickerDateNavigation: React.FunctionComponent<IProps> = ({
 				<div className="date-picker-nav-controls date-picker-nav-item date-picker-nav-item-expand">
 					<Button
 						aria-label={ariaLabels.buttonPreviousMonth}
+						className="nav-btn nav-btn-monospaced"
```

Comment:
Hey @pat270 I noticed some differences, we are adding `nav-btn-monospaced` but we have enabled `monospaced`, at least removing `btn-monospaced` I can not see any differences, I think we can disable this property of the buttons. I also noticed that `btn-sm` causes a small change, it apparently makes the icons a little smaller, would that be intentional?

Reply:
It's actually not supposed to have `btn-sm btn-monospaced`. I just left them in because there's a chance some projects might be depending on those classes. If you want them gone, I can remove them.

Reply:
Hmm makes sense. Anyway, I think it would be worth removing it and checking the Portal if it is used and replacing it otherwise it can stay a long time here waiting for the best day to remove it. Does it make sense to you?

Reply:
That's good with me. I removed and pushed changes up.